app/views/govuk_publishing_components/components/_accessible_autocomplete.html.erb

```diff
@@ -13,15 +18,20 @@
       }.merge(label.symbolize_keys)
     %>
 
-    <div class="gem-c-accessible-autocomplete" data-module="accessible-autocomplete">
+    <%= tag.div class: classes, data: { module: "accessible-autocomplete" } do %>
+      <% if multiple %>
+        <span class="govuk-hint app-c-autocomplete__multiselect-instructions">To select multiple items in a list, hold down Ctrl (PC) or Command (Mac) key.</span>
```

Comment:
Why is `Control` abbreviated for PC but `Command` for Mac isn't ?
Text needs translating too.

Reply:
Thanks - text changed and translated.